.github/dependabot.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+
+  - package-ecosystem: npm
```

Comment:
`package-ecosystem` needs to be changed to yarn, to ensure that it reads correct lockfile.
Other than that, nice! Update and I will merge.

Reply:
Not logical but npm value is used for both npm and yarn https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

Reply:
Oh, you are correct! Nice, approved!